projects/vdk-core/tests/functional/run/test_run_notifications.py

```diff
@@ -18,7 +19,7 @@
 
 def __get_smtp_env(smtpd: SMTPDFix):
     # https://github.com/bebleo/smtpdfix#using
-    smtpd.config.host = "127.0.0.1"
+    # smtpd.config.host = "127.0.0.1"
```

Comment:
Nit: can you please remove this? I forgot it's in there.

Reply:
ready